### PR TITLE
BIP 102: update to match implementation

### PR DESCRIPTION
**Pull request by jgarzik**
Contains latest ISM changes.

### CODE REVIEW

File: bip-0102.mediawiki [hunk 1]
```diff
@@ -13,28 +13,30 @@ Simple, one-time increase in total amount of transaction data permitted in a blo
...
+# Continue current economic policy.
+# Exercise hard fork network upgrade.
 
 ==Specification==
 
-# Maximum block size permitted to be valid is 1MB.
-# Increase this maximum to 2MB as soon as 75% of the last 1,000 blocks have signaled support.
+# MAX_BLOCK_SIZE increased to 2,000,000 bytes at trigger point.
 # Increase maximum block sigops by similar factor, preserving SIZE/50 formula.
+# Each 10-minute segment thereafter increases MAX_BLOCK_SIZE by 20 bytes.
```
Comment (MarcoFalke) [inline bip-0102.mediawiki]: People referring to BIP 102 have "bump to 2 MB once" in mind. I guess this needs a new BIP number?

**Comment by btcdrak** [comment]
Strong NAK. This is a fundamental change to a long standing BIP proposal. Allowing such a radical change would be confusing to people who has already backed BIP102. There should be a separate BIP proposal.

**Comment by luke-jr** [comment]
This is still Draft, so technically @jgarzik should be able to make any changes he wants, but I agree with the general sentiment here and strongly encourage not changing the BIP in such a very significant manner considering real-world use of the "BIP 102" concept in discussion.

**Comment by btcdrak** [comment]
The difficulty is that "BIP102" has been is already widely understood as a flat 2MB increase and has had a concrete PR open for several months. This change is a bit like tacking on defence spending onto a well oiled healthcare bill.

**Comment by jameshilliard** [comment]
NACK major changes like this should use a different BIP number, this will cause major confusion in regards to statements people have made in support of a particular proposal or against a proposal. For example the BIP100 tagging by miners was in support of the original draft proposal but NOT the subsequent changes made afterwards which made significant modifications to the cap decrease threshold in addition to the removal of the 32MB ceiling. I think non-trivial changes to draft BIP's should not be allowed without creating a new BIP number even if the proposal has not been officially submitted to this repo in order to ensure that people's statements in support of a particular proposal are not ambiguous.

**Comment by jgarzik** [comment]
Updated back to fixed one-time bump.

File: bip-0102.mediawiki [hunk 2]
```diff
@@ -13,28 +13,29 @@ Simple, one-time increase in total amount of transaction data permitted in a blo
...
+# Continue current economic policy.
+# Exercise hard fork network upgrade.
 
 ==Specification==
 
-# Maximum block size permitted to be valid is 1MB.
-# Increase this maximum to 2MB as soon as 75% of the last 1,000 blocks have signaled support.
+# MAX_BLOCK_SIZE increased to 2,000,000 bytes at trigger point.
 # Increase maximum block sigops by similar factor, preserving SIZE/50 formula.
+# Trigger:  (1) Block time 00:00:00 on flag day, AND (2) 75% of the last 1,000 blocks have signaled support.
```
Comment (jameshilliard) [inline bip-0102.mediawiki]: Shouldn't this be 95%?

Comment (btcdrak) [inline bip-0102.mediawiki]: Agreed, 95%

Comment (jgarzik) [inline bip-0102.mediawiki]: Updated

**Comment by jgarzik** [comment]
Update to 95%

**Comment by luke-jr** [comment]
@jgarzik I assume you will merge this yourself when ready?

**Comment by luke-jr** [comment]
Does anyone (especially @jgarzik ) object to BIP 102 turning into "Bitcoin Classic"'s hardfork proposal? In particular, they seem to want miner voting and add some new limits (to prevent DoS attacks; generally agreed to be necessary for size increases). I need to evaluate whether a new BIP would be redundant with BIP 102, or whether it would be best as a modification to it.

@jgarzik In addition to the above, would you be okay with considering an additional Champion/Author working on Classic (if applicable - I don't know how involved you are with their project)?

**Comment by jgarzik** [comment]
BIP 102 is not suddenly morphed into a tracking BIP for whatever Bitcoin Classic does.  That just creates additional confusion and a moving target.

We can assign a new BIP number to whatever Bitcoin Classic does.

**Pull request merged**